crates/core_arch/src/arm/crypto.rs

```diff
@@ -40,6 +54,7 @@ use stdarch_test::assert_instr;
 /// AES single round encryption.
 #[inline]
 #[target_feature(enable = "crypto")]
+#[cfg_attr(target_arch = "arm", target_feature(enable = "v8"))]
```

Comment:
Is the `v8` feature actually required here? Does it work without this feature?

Reply:
> Is the `v8` feature actually required here? Does it work without this feature?

Yes, `v8` feature is required since this is ARMv8 feature.  If this is `v7` or nothing, this causes `LLVM ERROR: Cannot select: intrinsic %llvm.arm.neon.aese` .